pom.xml

```diff
@@ -21,303 +21,304 @@
+                                </requireJavaVersion>
+                            </rules>
+                            <fail>true</fail>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>ban-known-bad-maven-versions</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireMavenVersion>
+                                    <version>[3.0.5,)</version>
+                                    <message>Maven minimal expected version is 3.0.5.</message>
+                                </requireMavenVersion>
+                            </rules>
+                        </configuration>
+                    </execution>
+                </executions>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>extra-enforcer-rules</artifactId>
+                        <version>1.0-beta-9</version>
+                    </dependency>
+                </dependencies>
+            </plugin>
+        </plugins>
+    </build>
 
-  <distributionManagement>
-    <site>
-      <id>velocity.apache.org</id>
-      <url>scpexe://people.apache.org/www/velocity.apache.org/engine/devel/
-      </url>
-    </site>
-  </distributionManagement>
+    <distributionManagement>
+        <site>
+            <id>velocity.apache.org</id>
+            <url>scpexe://people.apache.org/www/velocity.apache.org/engine/devel/
+            </url>
+        </site>
+    </distributionManagement>
 
-  <scm>
-    <connection>scm:svn:http://svn.apache.org/repos/asf/velocity/engine/trunk</connection>
-    <developerConnection>scm:svn:https://svn.apache.org/repos/asf/velocity/engine/trunk</developerConnection>
-    <url>http://svn.apache.org/viewvc/velocity/engine/trunk</url>
-  </scm>
+    <scm>
+        <connection>scm:svn:http://svn.apache.org/repos/asf/velocity/engine/trunk</connection>
+        <developerConnection>scm:svn:https://svn.apache.org/repos/asf/velocity/engine/trunk</developerConnection>
+        <url>http://svn.apache.org/viewvc/velocity/engine/trunk</url>
+    </scm>
 
-  <issueManagement>
-    <system>JIRA</system>
-    <url>${jira.browse.url}/VELOCITY</url>
-  </issueManagement>
-  <modules>
-    <module>velocity-engine-core</module>
-    <module>velocity-engine-examples</module>
-    <module>velocity-engine-scripting</module>
-  </modules>
+    <issueManagement>
+        <system>JIRA</system>
+        <url>${jira.browse.url}/VELOCITY</url>
+    </issueManagement>
+    <modules>
+        <module>velocity-engine-core</module>
+        <module>velocity-engine-examples</module>
+        <module>velocity-engine-scripting</module>
+        <module>velocity-custom-parser-example</module>
```

Comment:
You reformatted the entire POM whereas the change is just one line. Can you revert that? It makes tracking changes hard.

Reply:
All reformatting and plugins reordering has been done in a single commit (svn rev 1860484). Here is [the diff of changes before pom reorganization], and [the diff of subsequent changes](https://republicate.com/scratch/after_reorg.html).